tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```
@@ -286,9 +286,10 @@ spec:
           memory: 256Mi
       image: quay.io/enterprise-contract/ec-cli:snapshot
       onError: continue  # progress even if the step fails so we can see the debug logs
-      command: [cat]
+      command: [sh, -c]
       args:
-        - "$(params.HOMEDIR)/report-json.json"
+        # Format the JSON output to wrap lines at 8000 characters for readability
+        - "jq . $(params.HOMEDIR)/report-json.json | awk '{gsub(/^ +/, \"\"); acc += length; if (acc >= 8000) { printf \"\n\"; acc=length } printf $0 }'"
```

Looking at the error in the acceptance test made be thing perhaps you have to escape the \ char in the \n. Not sure though...

```suggestion
        - "jq . $(params.HOMEDIR)/report-json.json | awk '{gsub(/^ +/, \"\"); acc += length; if (acc >= 8000) { printf \"\\n\"; acc=length } printf $0 }'"
```